kiwi/runtime_config.py

```diff
@@ -309,6 +309,25 @@ def get_iso_tool_category(self):
             )
             return Defaults.get_iso_tool_category()
 
+    def get_iso_media_tag_tool(self):
+        """
+        Return media tag tool used to checksum iso images
+
+        iso:
+          - media_tag_tool: checkmedia
+
+        if no configuration exists the default media tagger
+        from the Defaults class is returned
+
+        :return: A name
+
+        :rtype: str
+        """
+        iso_media_tag_tool = self._get_attribute(
+            element='iso', attribute='media_tag_tool'
+        )
+        return iso_media_tag_tool or Defaults.get_iso_media_tag_tool()
```

Comment:
I'd suggest that we fail if the result is not in `['checkmedia', 'isomd5sum']`

Reply:
I tried to do this earlier, but I couldn't get the unit test coverage to be complete for it, so I dropped the logic since we don't do this for several other ones either...

Reply:
We don't fail, we select the default if it's not a valid choice. But I've put the logic back.